pkg/config/config.go

```diff
@@ -137,6 +143,7 @@ func ReadFromCommandLineFlags() {
 		"The path to the config file (i.e poseidon_cfg) without filename or extension, supported extensions/formats are Yaml, Json")
 	flag.BoolVar(&config.EnablePprof, "enablePprof", false, "Enable runtime profiling data via HTTP server. Address is at client URL + \"/debug/pprof/\"")
 	flag.StringVar(&config.PprofAddress, "pprofAddress", "0.0.0.0:8989", "Address on which to collect runtime profiling data,default to set for all interfaces ")
+	pflag.StringVar(&config.Port, "port", "9292", "The port that the poseidon's http service runs on,default to set for all interfaces")
```

Comment:
I would prefer MetricsBindAddress than port.

Reply:
Plus, I would suggest default metrics(and probably healthz in the future) reuse the same port - 8989.

Reply:
agree with u, commit submitted